### PR TITLE
Gh pages test

### PR DESCRIPTION
**Pull request by aghtetnyein**
**This is my portfolio deployed with github pages. Can you please check and help me what things should I need to fix?**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,59 @@
...
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Sa Aung Htet Nyein's Microverse Portfolio
+
+> My personal portfolio and the first project of Module 1 at Microverse.
+
+## Live Demo
+
+[To my Portfolio Live Demo](https://aghtetnyein.github.io)
+
```
Comment (ghost) [inline README.md:10]: - [ ] Remember that this link is supposed to display your actual portfolio page.
### Your display
image

- [ ] Would you please follow these [steps](https://www.freecodecamp.org/news/publish-your-website-netlify-github/) to deploy your website

**Review by frank1738 — CHANGES_REQUESTED** [review]
Hi  @aghtetnyein , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

YesYouAreGIF (2)


## Highlights

✅ Correct use of Github workflow
✅ No linter error
✅ README is okay


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (aghtetnyein) [reply README.md:10]: Thanks, much to you @frank1738 
I think the actual portfolio page will appear after I merge this branch to the main and so should I merge it to view the live Demo?

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @aghtetnyein👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (ghost) [reply README.md:10]: - [ ] @aghtetnyein.  You can deploy the current branch. you don't need to merge

**Pull request merged**